<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QMainWindow" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="commandLinkButton_setPwd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>177</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置用户密码</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_setMainControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>177</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置第一人称控制</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_addCharacter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>177</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加角色控制器</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_removeCharacter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>177</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除角色控制器</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>761</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_setPosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>177</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置角色控制器位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_db_put">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>写入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_db_get">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_db_key">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>数据库操作</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>250</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>区块坐标</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_chunk_x">
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_chunk_y">
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_getNearOnline">
         <property name="text">
          <string>查询附近在线</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_getNearUser">
         <property name="text">
          <string>查询附近玩家</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
